app/views/shared/_footer.html.erb

```diff
@@ -5,6 +5,10 @@
     <%= render "shared/footer/contact" %>
   </div>
 
+  <div class="site-mode-link-container">
+    <p><%= link_to_unless(lite_mode?, t("footer.site_mode"), url_for_current_path_with_subdomain(subdomain: :lite)) {} %></p>
```

Comment:
```suggestion
    <p><%= link_to(t('footer.site_mode'), url_for_current_path_with_subdomain(subdomain: :lite)) unless lite_mode?  %></p>
```

Reply:
IMO, it reads cleaner with the conditional at the end, instead of the dangling empty hash. What do you think?